src/components/VideoThumbnail/VideoThumbnail.scss

```diff
@@ -77,8 +78,8 @@ $start-button-size: 60px;
   position: absolute;
   bottom: 0;
   width: 100%;
-  background-color: rgba(color('white'), 0.9);
-  height: spacing(extra-tight);
+  background-color: color('white');
```

Comment:
Did you mean to remove the semi-transparency of the progress bar?

Reply:
Yes, this change is intentional. Had to make it last minute to match implementation across all platforms (iOS and Android)